bcda/testUtils/client.go

```diff
@@ -18,8 +18,8 @@ type BlueButtonClient struct {
 	MBI  *string
 }
 
-func (bbc *BlueButtonClient) GetExplanationOfBenefit(patientID, jobID, cmsID, since string, transactionTime time.Time) (*models.Bundle, error) {
-	args := bbc.Called(patientID)
+func (bbc *BlueButtonClient) GetExplanationOfBenefit(patientID, jobID, cmsID, since string, transactionTime, serviceDate time.Time) (*models.Bundle, error) {
```

Comment:
This was changed so we can verify that all arguments are called as expected. Before, we just verified the patientID argument.

Reply:
good idea